contracts/Liquidator/Liquidator.sol

```diff
@@ -22,6 +23,8 @@ interface IVBep20 is IVToken {
         uint256 repayAmount,
         IVToken vTokenCollateral
     ) external returns (uint256);
+
+    function redeem(uint256 redeemTokens) external returns (uint256);
```

Comment:
`redeem` is still available in `vBNB`. I would move it to the `IVToken` interface